cfme/tests/automate/test_common_methods.py

```diff
@@ -390,3 +390,67 @@ def _check():
 
     wait_for(lambda: _check, fail_func=view.browser.refresh, timeout=600, delay=5,
              message="Waiting for vm folder name to appear")
+
+
+@pytest.mark.tier(1)
+@pytest.mark.meta(automates=[1574444])
+@pytest.mark.provider([VMwareProvider], selector=ONE)
+def test_list_of_diff_vm_storages_via_rails(appliance, setup_provider, provider, testing_vm,
+                                            custom_instance):
+    """
+    Bugzilla:
+        1574444
+
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        caseimportance: medium
+        caseposneg: positive
+        testtype: functional
+        startsin: 5.9
+        casecomponent: Automate
+        testSteps:
+            1. vmware = $evm.vmdb('ems').find_by_name('vmware 6.5 (nested)') ;
+            2. vm = vmware.vms.select { |v| v.name == 'ghubale-cfme510' }.first ;
+            3. vm.storage
+            4. vm.storages
+        expectedResults:
+            1.
+            2.
+            3. Returns only one storage
+            4. Returns available storages
+    """
+    list_storages = dedent(
+        f'vmware = $evm.vmdb("ems").find_by_name("{provider.name}")\n'
```

Comment:
As we are using `dedent`, why not use full script?

Reply:
I agree @digitronik 
- But I am not able to format this line using **fstring** or **.format**(```vm = vmware.vms.select {|v| v.name == "{testing_vm.name}"}.first```) because of extra ```{}```.
- Hence I modified this script in this way. Please let me know if you have a solution for formatting this script.

Reply:
This can be done in either with a double `{` character.

```
In [8]: storages = (f'vmware = $evm.vmdb("ems").find_by_name("{provider_name}")\n' 
   ...: f'vm = vmware.vms.select {{|v| v.name == "{test_vm_name}"}}.first\n' 
   ...: )                                                                                                                     

In [9]: storages                                                                                                              
Out[9]: 'vmware = $evm.vmdb("ems").find_by_name("test_provider_name")\nvm = vmware.vms.select {|v| v.name == "test_virtual_machine"}.first\n'
```

The single braced expressions are evaluated, the double braced characters result in a single brace in the resulting string.

https://docs.python.org/3.7/reference/lexical_analysis.html#formatted-string-literals

First paragraph below the code block.

Reply:
That said, the formatting change can be a followup, I won't block merging on this. The use of dedent and multiple lines makes it clear what is being evaluated and what is not.